### PR TITLE
Add documentation for `ploomber-cloud labels --sync`

### PR DESCRIPTION
**Pull request by KJSanchez**
documentation for https://github.com/ploomber/cli/issues/94

----
📚 Documentation preview 📚: https://ploomber-doc--261.org.readthedocs.build/en/261/

### CODE REVIEW

**Review by bryannho — CHANGES_REQUESTED** [review]
@KJSanchez Please also add a note in `doc/user-guide/managing-apps.md` under the Labels section, linking users to the CLI guide. You can do it as a tip:
````
```{tip}
You can also add labels to your app through the [CLI](link-to-CLI-labels-section)
```
````

File: doc/user-guide/cli.md [hunk 1]
```diff
@@ -370,6 +370,14 @@ To view all labels added to your project simply run:
 ploomber-cloud labels
 ```
 
+### Synchronize labels
+
+To update your local config with the labels from the UI:
+
+```sh
+ploomber-cloud labels --sync
+```
```
[the remark this answers is not in the document]
Comment (KJSanchez) [reply doc/user-guide/cli.md:379]: https://github.com/ploomber/doc/pull/261/commits/555bd0eb25f9982fdcbaad0a323dc5da4ea81d69

**Comment by KJSanchez** [comment]
> @KJSanchez Please also add a note in `doc/user-guide/managing-apps.md` under the Labels section, linking users to the CLI guide. You can do it as a tip:
> 
> ```
> ```{tip}
> You can also add labels to your app through the [CLI](link-to-CLI-labels-section)
> ```
> ```

https://github.com/ploomber/doc/pull/261/commits/d8d518b0715aa6d9a325427db2417dafd412ce43

and relevant link to the preview app: https://ploomber-doc--261.org.readthedocs.build/en/261/user-guide/managing-apps.html

**Review by bryannho — DISMISSED** [review]

**Comment by bryannho** [comment]
@KJSanchez let's wait for @neelasha23 to approve this PR and the CLI PR, then we'll merge

File: doc/user-guide/managing-apps.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 # Managing applications
 
 ## Labels
-
+```{tip}
+You can add labels to your app through the [CLI](./cli.md#add-labels)
```
Comment (neelasha23) [inline doc/user-guide/managing-apps.md:5]: This can be added as a note at the end of the `Labels` section

Comment (KJSanchez) [reply doc/user-guide/managing-apps.md]: https://github.com/ploomber/doc/pull/261/commits/636f10183950535c600456653da2986448f17357

**Review by neelasha23 — APPROVED** [review]

**Review by bryannho — APPROVED** [review]

**Pull request merged**